chapter_convolutional-neural-networks/lenet.md

```diff
@@ -380,11 +380,21 @@ class TrainCallback(tf.keras.callbacks.Callback):  #@save
 
 #@save
 def train_ch6(net_fn, train_iter, test_iter, num_epochs, lr,
-              device=d2l.try_gpu()):
+              device=d2l.try_gpu(), mirrored=True):
```

Comment:
I feel it complicates the implementation, that's why I removed it before. Is there a simple way for TF to run on a single GPU? or we can remove the batch norm from scratch training.

Reply:
I just realized that there's a [OneDeviceStrategy](https://www.tensorflow.org/api_docs/python/tf/distribute/OneDeviceStrategy) we can use. I'll give it a try.

Reply:
Thanks